TwipsterApp/TwipsterMainWindow.xaml.cs

```diff
@@ -37,9 +52,39 @@ private void OnLogOutButtonClicked(object sender, RoutedEventArgs e)
             Close();
         }
 
+        private void OnShowAllPostsButtonClicked(object sender, RoutedEventArgs e)
+        {
+            using var context = new TwipsterDbContext();
+            PostGridRefresh(context);
+        }
+
+        private void OnShowSelectedUserPostsClicked(object sender, RoutedEventArgs e)
+        {
+            using var context = new TwipsterDbContext();
+            var selectedUserLogin = UsersGrid.SelectedItem as SelectedUserModel;
```

Comment:
Rename this variable. This variable is not describing selected user login. This variable is describing selected user.
Or you can get login in this line instead of getting in the next one.